testing/opendmarc/APKBUILD

```diff
@@ -0,0 +1,39 @@
+# Contributor: Hidde van der Heide <hvanderheide@nexuz.net>
+# Maintainer:
+pkgname=opendmarc
+pkgver=1.3.2
+pkgrel=1
+pkgdesc="OpenDMARC is a free open source software implementation of the DMARC specification"
+url="http://www.trusteddomain.org/opendmarc/"
+arch="all"
+license="Sendmail"
+depends=""
+makedepends="libmilter-dev"
+subpackages="$pkgname-doc"
```

Comment:
It installs some header files and static lib, so subpackage `$pkgname-dev` should be added.